### PR TITLE
docs: add trpc to examples

### PR DESCRIPTION
**Pull request by zorrolisto**
## Documentation / Examples

- [x] Make sure the linting passes by running `pnpm lint`
- [x] The examples guidelines are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

Nice to have nextjs up to date always!

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, I would like to defer to @KATT here. I wonder if tRPC already has an official example, and it would make sense to add a single README.md file here that points to their example. This would make sure that we are in sync with the latest recommendations by the tRPC team.

**Review by KATT — COMMENTED** [review]
I love how minimal this example is, @zorrolisto, do you wanna add it to https://github.com/trpc/trpc?

We could instead do a README.md with a few links:

- https://github.com/trpc/examples-next-prisma-starter
- https://github.com/trpc/examples-next-prisma-starter-websockets
- [... your minimal example if you add it 🙃]

**Comment by zorrolisto** [comment]
thanks 🙏, so I need to remove the directories and files, and only left one README.md with the links right? @KATT

**Comment by balazsorban44** [comment]
That's correct!

File: examples/with-trpc/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Next.js + TRPC
```
Comment (balazsorban44) [inline examples/with-trpc/README.md:1]:
```suggestion
# Next.js + tRPC
```

File: examples/with-trpc/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Next.js prisma starter
```
Comment (balazsorban44) [inline examples/with-trpc/README.md:3]:
```suggestion
## Next.js + tRPC + Prisma starter
```

File: examples/with-trpc/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- https://github.com/trpc/examples-next-prisma-starter
+
+## Next.js prisma starter websockets
```
Comment (balazsorban44) [inline examples/with-trpc/README.md:7]:
```suggestion
## Next.js + tRPC + Prisma starter with websockets
```

File: examples/with-trpc/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+- https://github.com/trpc/examples-next-prisma-starter-websockets
+
+## Next.js and TRPC minimal implementation
```
Comment (balazsorban44) [inline examples/with-trpc/README.md:11]:
```suggestion
## Next.js and tRPC minimal implementation
```

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**